opencog/nlp/aiml/import/aiml2psi.pl

```diff
@@ -545,17 +549,24 @@ sub process_tag
 
 	$text =~ /(.*?)<$tag>(.*?)<\/$tag>(.*)/;
 
-	$tout .= &process_aiml_tags($indent, $1);
+	push(@pt1, $1);
+	push(@pt2, $2);
+	push(@pt3, $3);
+
```

Comment:
Huh. I guess maybe this works, but a much simpler solution is possible: just make a copy -- so
```
my $first = $1;
my $middle = $2;
my $rest_of_it = $3;
$tout .= &process_aiml_tags($indent, $first);
...
$tout .= &process_aiml_tags($indent, $middle);
if ($rest_of_it ne "")
{
    $tout .= &process_aiml_tags($indent, $rest_of_it);
}
```

or just call it `my $t1, $t2, $t3,` etc.